lib/src/signed_video_h26x_auth.c

```diff
@@ -70,6 +70,22 @@ static const char *kAuthResultValidStr[SV_AUTH_NUM_SIGNED_GOP_VALID_STATES] = {"
     "SIGNATURE PRESENT", "NOT OK", "OK WITH MISSING INFO", "OK", "VERSION MISMATCH"};
 #endif
 
+/*
+ * Checks if linked hash validation is feasible at the start of validation.
+ * During the first validation slots, the hashes in |gop_info->linked_hashes| are all zeros,
+ * indicating that validation is not possible.
+ */
+static bool
+linked_hash_validation_is_not_feasible(signed_video_t *self)
+{
+  gop_info_t *gop_info = self->gop_info;
+  const size_t hash_size = self->verify_data->hash_size;
+  uint8_t linked_hashes[hash_size];
+  memset(linked_hashes, 0x00, sizeof(linked_hashes));
```

Comment:
I don't think you need to use memset. It should be possible to set values already at declaration. Try
`const uint8_t linked_hash[MAX_HASH_SIZE] = {0};`